### PR TITLE
INTER/TYPE11 - Compatibility Igap=3 Iremgap=2

### PR DESCRIPTION
**Pull request by bertrand-altair**
Allow compatibilty between Igap=3 and Iremgap=2

### CODE REVIEW

File: starter/source/interfaces/inter3d1/i11remlin.F [hunk 1]
```diff
@@ -47,6 +47,7 @@ SUBROUTINE I11REMLINE(
       my_real
      .        X(3,*),GAP_S(*),GAP_M(*),GAP,DRAD,GAPMIN
       my_real , INTENT(IN) :: DGAPLOAD
+      my_real , INTENT(IN) :: GAP_S_L(*), GAP_M_L(*)
```
Comment (sebastienVilleneuve) [inline starter/source/interfaces/inter3d1/i11remlin.F:50]: @bertrand-altair 

dimension of GAP_S_L &  GAP_M_L are missing ( you are using '*') , could you correct this please ?

Thanks,
Sébastien

File: starter/source/interfaces/inter3d1/i11remlin.F [hunk 2]
```diff
@@ -279,13 +280,14 @@ SUBROUTINE I11REMLINE(
               ENDIF
             ENDDO
             KREMNODE(I+1) = KREMNODE(I) + CPT1
-          ELSEIF (IGAP == 1) THEN 
+          ELSE
```
Comment (sebastienVilleneuve) [inline starter/source/interfaces/inter3d1/i11remlin.F:283]: here you also change the behavior for Iremgap=2 + Igap = 2 , is it the behavior you expect ?

**Review by sebastienVilleneuve — CHANGES_REQUESTED** [review]
@bertrand-altair 

dimension of GAP_S_L &  GAP_M_L are missing ( you are using '*') , could you correct this please ?

Thanks,
Sébastien

File: starter/source/interfaces/inter3d1/i11remlin.F (hunk 2 above)
Comment (bertrand-altair) [reply starter/source/interfaces/inter3d1/i11remlin.F:283]: Done

File: starter/source/interfaces/inter3d1/i11remlin.F (hunk 1 above)
Comment (bertrand-altair) [reply starter/source/interfaces/inter3d1/i11remlin.F]: Done

**Review by sebastienVilleneuve — APPROVED** [review]

**Pull request merged**